controllers/resources/manager.go

```diff
@@ -161,7 +161,8 @@ func (m *manager) CreateRemediationCR(remediationCR *unstructured.Unstructured,
 	m.log.Info("Attempting to obtain Node Lease",
 		"Node name", remediationCR.GetName())
 
-	isLeaseObtained, requeue, err := m.leaseManager.ObtainNodeLease(remediationCR, nhc)
+	requeue, err := m.leaseManager.ObtainNodeLease(remediationCR, nhc)
+	isLeaseObtained := err == nil
```

Comment:
I think we can just remove the `isLeaseObtained` var. After the error check below we know it was obtained, so the next `if` can be removed and the return value just be `true`